src/cclib/parser/daltonparser.py

```diff
@@ -250,7 +249,7 @@ def extract(self, inputfile, line):
                 #self.skip_lines(inputfile, ['b', 'basisname', 'b'])
                 line = next(inputfile)
                 line = next(inputfile)
-                self.metadata["basis_set_name"] = re.findall(r'"([^"]*)"', line)[0]
+                self.metadata["basis_set_name"] = line.split()[4][1:-1]
```

Comment:
What are trying to remove here, apostrophes or something? If so, please use `line.split()[4].strip(...)` or `.replace(...)`.